.github/workflows/ci.yml

```
@@ -0,0 +1,100 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, feat/coding-standards ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)